README.md

```diff
@@ -74,7 +91,7 @@ Available Commands:
   help        Help about any command
   init        Write default configuration to file
   list        List channels, claims in progress, etc
-  serve       
+  serve       (default) option
```

Comment:
1. Text above is actually copied from ```./build/snetd-linux-amd64 --help``` output, so I would suggest adding description for serve command instead. 
2. May be it makes sense to rename ```serve``` to ```start``` command as it should be more obvious.
3. I think it is not required to keep ```serve``` (or ```start```) as default behavior, I think now it is time when we can replace all usages by explicit ```snetd-linux-amd64 start``` especially if it is confusing.

Reply:
Lets track the default behaviour in a separate issue that I will create now. From a readme perspective lets release this now as it reflects current behaviour and fix it along with the issue

Reply:
for now , re worded to "Is the default option which starts the Daemon"

Reply:
Sure, I didn't mean that 2 and 3 should be done in this PR.
Thanks, @raamb, for raising issue on this.